apps/music/bower.json

```diff
@@ -10,13 +10,13 @@
   ],
   "dependencies": {
     "bridge": "gaia-components/bridge#master",
-    "gaia-fast-list": "gaia-components/gaia-fast-list#~0.2.2",
-    "gaia-header": "gaia-components/gaia-header#~0.9.5",
+    "gaia-fast-list": "gaia-components/gaia-fast-list#master",
```

Comment:
I'm not super-familiar with bower, but doesn't this mean that every time we update our bower deps, we'll fetch the latest code from gaia-fast-list/master? That doesn't seem like a good idea, since we could end up pulling in a newer gaia-fast-list than we intend and potentially cause new bustage.